### PR TITLE
Update Translation README.md for workflow

### PR DESCRIPTION
**Pull request by louie-tsai**
## Description

Added a workflow for VideoQnA following what ChatQnA has. [ChatQnA flow](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA#architecture-and-deploy-details)

## Issues

 `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

na

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: Translation/README.md [hunk 1]
```diff
@@ -6,6 +6,57 @@ Translation architecture shows below:
...
+    subgraph UserInterface[" User Interface "]
+        direction LR
+        a([User Input Query]):::orchid
+        UI([UI server<br>]):::orchid
+    end
+
+
+    LLM_gen{{LLM Service <br>}}
+    GW([Translation GateWay<br>]):::orange
+    NG([Nginx MicroService]):::blue
```
Comment (lvliang-intel) [inline Translation/README.md:43]: Why nginx is in the front of the user input?

Comment (louie-tsai) [reply Translation/README.md:43]: I saw a nginx microservice as a proxy server in docker compose, and UI server depends it.
https://github.com/opea-project/GenAIExamples/blob/main/Translation/docker_compose/intel/hpu/gaudi/compose.yaml#L77

should I remove it or put it somewhere else in the workflow?

Comment (louie-tsai) [reply Translation/README.md:43]: updated them accordingly.

**Comment by louie-tsai** [comment]
@lvliang-intel  @lkk12014402 
Could you help to review and approve the PR?

**Review by lvliang-intel — APPROVED** [review]

**Comment by louie-tsai** [comment]
@lkk12014402 
possible to help review the PR?

**Pull request merged**